### DISCUSSION TITLE
Push RestDesk client with ID server and API Key to Domain "DC"

### DISCUSSION THREAD

**Discussion by gamal3330** [rustdesk/rustdesk#7977] [General]
Daer Rust Team ,

I want to push RustDesk.exe with ID Server and API Key to domain .

**Comment by rustdesk** [comment]
https://twitter.com/rustdesk/status/1769171628426944539

**Reply by gamal3330** [reply]
I used free RustDesk without RustDesk Console

**Reply by DMSnz** [reply]
Then look at docs, there are guides for using TRMM to deploy, but the docs are essentially a powershell script which you can then use to push out via GPOs on your domain, this is exactly how I have done it so far.
https://docs.tacticalrmm.com/3rdparty_rustdesk/
